app/controllers/mobile_controller.rb

```diff
@@ -23,6 +23,67 @@ def logout
     end
   end
   
+  def add_payment_method
+    if authenticate
+      payment_type = params[:payment_type].downcase
+      if payment_type == "cash"
+        @user.payment_methods.create!(payment_type: payment_type)
```

Comment:
how are you guys enforcing SSL? I think there was 
```ruby
 config.force_ssl = true
```
https://github.com/tobmatth/rack-ssl-enforcer is the first thing that came up when I googled. It's from 2015 w/ 29 contributors.